src/dolphin/install/macos.ts

```diff
@@ -7,17 +7,19 @@ import { extractDmg } from "./extractDmg";
 export async function installDolphinOnMac({
   assetPath,
   destinationFolder,
+  skipUserFolderBkp,
   log = console.log,
 }: {
   assetPath: string;
   destinationFolder: string;
+  skipUserFolderBkp: boolean;
```

Comment:
```suggestion
  shouldBackupUserFolder?: boolean;
```
This is an exposed type so we shouldn't be using short-names. Can we also change the prop so that we have it true when we want to backup instead of having it true to skip back up and then you can just set the default appropriately.